students/makarov-hw-2/1-calculator/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Abacus</title>
+</head>
+<body>
+    <article class="calc">
+        <div class="calc__keyboard">
+            <button onclick="calculate(plus, getNum)" class="calc__btn">+</button>
+            <button onclick="calculate(minus, getNum)" class="calc__btn">-</button>
+            <button onclick="calculate(multiplication, getNum)" class="calc__btn">*</button>
+            <button onclick="calculate(division, getNum)" class="calc__btn">/</button>
+        </div>
+    </article>
+    <style>
+        *{
+            box-sizing: border-box;
+        }
+        body{
+            margin: 0;
+            width: 100vw;
+            max-width: 100%;
+            height: 100vh;
+            max-height: 100%;
+        }
+        .calc{
+            position: absolute;
+            top: 50%;
+            left: 50%;
+            transform: translate(-50%, -50%);
+            width: 500px;
+            height: 500px;
+            border: 1px solid salmon;
+        }
+        .calc__keyboard{
+            display: flex;
+            flex-wrap: wrap;
+            height: 500px;
+        }
+        .calc__btn{
+            display: block;
+            width: 50%;
+            height: 50%;
+            font-size: 150px;
+        }
+    </style>
+    <script>
+        function calculate(solverFunc, inputFunc) {
+            const a = inputFunc('Enter first num');
+            const b = inputFunc('Enter second num');
+            const result = solverFunc(a,b);
+            console.log(result);
+        };
+
+        function getNum(str) {
+            let a = +prompt(str, 0);
+            if(!isNaN(a) && isFinite(a)){
+                return a;
+            }
+            return getNum(str);
+        };
+
+        const plus = (a, b) => (a + b).toFixed(6);
+        const minus = (a, b) => (a - b).toFixed(6);
+        const multiplication = (a, b) => (a * b).toFixed(6);
+        const division = (a, b) => b === 0 ? 'The universe is falling apart because of you' : (a / b).toFixed(6);
```

Comment:
const division = (a, b) => !b ? 'The universe is falling apart because of you' : (a / b).toFixed(6);